chrome/install.xul

```diff
@@ -66,6 +66,16 @@
 		</description>
 	</wizardpage>
 	
+    <wizardpage pageid="jdk-required" onpageshow="jreRequiredPageShown()"
+	label="Install Required Components"
+	next="openoffice-installations">
+		<description>
+			Zotero LibreOffice Integration on MacOS requires the Java 
```

Comment:
"macOS", and we should probably drop the "integration" stuff as much as possible. "The Zotero LibreOffice Plugin on macOS requires…"